catalogue/webapp/pages/item.js

```diff
@@ -1,29 +1,120 @@
+import Paginator from '@weco/common/views/components/Paginator/Paginator';
 type Props = {|
   workId: string,
+  sierraId: string,
+  manifest: {},
+  pageIndex: number,
+  itemsLocationsLocationType: string[],
+  workType: string[],
 |};
 
-const ItemPage = ({ workId }: Props) => (
-  <PageLayout
-    title={''}
-    description={''}
-    url={{ pathname: `/works/${workId}/items` }}
-    openGraphType={'website'}
-    jsonLd={{ '@type': 'WebPage' }}
-    siteSection={'works'}
-    imageUrl={'imageContentUrl'}
-    imageAltText={''}
-    hideNewsletterPromo={true}
-  >
-    <h1>{workId}</h1>
-  </PageLayout>
-);
+const ItemPage = ({
+  workId,
+  sierraId,
+  manifest,
+  pageIndex,
+  itemsLocationsLocationType,
+  workType,
+}: Props) => {
+  const canvases = manifest.sequences[0].canvases;
+  const currentCanvas = canvases[pageIndex];
+  const service = currentCanvas.thumbnail.service;
+  const urlTemplate = iiifImageTemplate(service['@id']);
+  const title = manifest.label;
+  const largestSize = service.sizes[service.sizes.length - 1];
+
+  return (
+    <PageLayout
+      title={''}
+      description={''}
+      url={{ pathname: `/works/${workId}/items` }}
+      openGraphType={'website'}
+      jsonLd={{ '@type': 'WebPage' }}
+      siteSection={'works'}
+      imageUrl={'imageContentUrl'}
+      imageAltText={''}
+      hideNewsletterPromo={true}
+    >
+      <Layout12>
+        <h1>{title}</h1>
+
+        <Paginator
+          currentPage={pageIndex + 1}
+          pageSize={1}
+          totalResults={canvases.length}
+          link={itemUrl({
+            workId,
+            query: null,
+            page: pageIndex - 1,
+            workType,
+            itemsLocationsLocationType,
+            sierraId,
+          })}
+          onPageChange={async (event, newPage) => {
+            event.preventDefault();
+
+            const link = itemUrl({
+              workId,
+              query: null,
+              workType,
+              itemsLocationsLocationType,
+              page: newPage,
+              sierraId,
+            });
+
+            Router.push(link.href, link.as).then(() => window.scrollTo(0, 0));
+          }}
+        />
+        <img
+          width={largestSize.width}
+          height={largestSize.height}
+          src={urlTemplate({
+            size: `${largestSize.width},${largestSize.height}`,
+          })}
+        />
+      </Layout12>
+    </PageLayout>
+  );
+};
 
 ItemPage.getInitialProps = async (ctx: Context): Promise<Props> => {
-  const { workId } = ctx.query;
-  return { workId };
+  const { workId, sierraId, page } = ctx.query;
+  const pageIndex = page ? parseInt(page, 10) - 1 : 0;
+  const manifest = await (await fetch(
+    `https://wellcomelibrary.org/iiif/${sierraId}/manifest`
+  )).json();
+  const itemsLocationsLocationType =
+    'items.locations.locationType' in ctx.query
+      ? ctx.query['items.locations.locationType'].split(',')
+      : ['iiif-image'];
```

Comment:
We should try get the context stuff in, this might be a good place to test it without breaking anything.